resource-managers/mesos/src/test/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerBackendUtilSuite.scala

```diff
@@ -50,4 +52,20 @@ class MesosSchedulerBackendUtilSuite extends SparkFunSuite {
     assert(params.get(2).getKey == "c")
     assert(params.get(2).getValue == "3")
   }
+
+  test("ContainerInfo respects Docker network configuration") {
+    val networkName = "test"
+    val conf = new SparkConf()
+    conf.set(config.CONTAINERIZER, "docker")
+    conf.set(config.EXECUTOR_DOCKER_IMAGE, "image")
+    conf.set(config.NETWORK_NAME, networkName)
+
+    val containerInfo = MesosSchedulerBackendUtil.buildContainerInfo(conf)
+
+    assert(containerInfo.getDocker.getNetwork == DockerInfo.Network.USER)
+    val params = containerInfo.getDocker.getParametersList
+    assert(params.size() == 1)
```

Comment:
Very small nit, but if you make another round of changes, make the asserts use `===`

Reply:
Thanks, updated the test.